insights/config/specs.py

```diff
@@ -20,9 +20,9 @@
                                     SimpleFileSpec("sos_commands/filesys/blkid")]),
     "bond"                      : PatternSpec(r"proc/net/bonding/bond.*"),
     "brctl_show"                : CommandSpec("/usr/sbin/brctl show"),
-    "candlepin_error_log"       : SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/error.log", large_content=True),
     "candlepin.log"             : First([SimpleFileSpec(r"var/log/candlepin/candlepin.log", large_content=True),
                                     SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/candlepin.log", large_content=True)]),
+    "candlepin_error_log"       : SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/error.log", large_content=True),
```

Comment:
Better user NoneGroup to wrap this SimpleFileSpec, which to indicate the client tool does not try to collect this file.